roles/common/tasks/main.yml

```diff
@@ -18,6 +18,33 @@
   tags:
     - common
 
+- name: preserve hostname
+  sudo: yes
+  lineinfile:
+    dest: /etc/cloud/cloud.cfg
```

Comment:
just b/c I didn't realize it was an option. that would be better. I'll give it a shot